workshops/templates/workshops/workshop_issues.html

```diff
@@ -36,7 +36,7 @@
         {% if event.mailto_ %}
           <a href="mailto:{{event.mailto_}}?subject={% filter urlencode %}Missing location for workshop {{event.get_ident}}{% endfilter %}&body={% filter urlencode %}Hi,
 
-Can you please send us the location where the {{ event.get_ident }} workshop took place?
+Can you please send us the location where the {{ event.get_ident }} workshop took place? We need a country, address, venue, and latitude/longitude.
```

Comment:
@maneesha after introduction of this PR, the list of workshops without location info is long (~40?). It's because we changed some rules:

* start date, URL, country, venue, address, latitude and longitude must be in place for the workshop to be considered published
* we're not looking for upcoming workshops only, but for all we ever had (this mainly introduces the big number of workshops with missing all location data)

Why all these changes? Because we didn't have clear rules in all places (see the list below) + because of https://github.com/swcarpentry/amy/issues/547 + because of related https://github.com/swcarpentry/site/issues/1130:

* API endpoint published events
* workshop issues page
* dashboard

So summing up: with this PR we're cleaning up some a small mess that was covering loops in our data and logic.